maquetado/index.html

```diff
@@ -9,22 +9,25 @@
   <link rel="stylesheet"  type="text/css" href="application.css">
 </head>
 <body>
-  <div class="book-container">
-    <div class="row">
-      <h1 class="book-title bold height">Título del libro</h1>
-      <h2 class="book-gender height">(género)</h2>
-    </div>
-    <div class="row">
-      <h3 class="book-detail-name bold height">Autor del libro:</h3>
-      <h3 class="book-detail-value height">Nombre del autor del libro</h3>
-    </div>
-    <div class="row">
-      <h3 class="book-detail-name bold height">Editorial:</h3>
-      <h3 class="book-detail-value height">Nombre de la editorial</h3>
-    </div>
-    <div class="row">
-      <h3 class="book-detail-name bold height">Año de publicación:</h3>
-      <h3 class="book-detail-value height">Año de publicación</h3>
+  <div class="book-container row">
+    <img src="./assets/book-cover.png" class="book-cover">
```

Comment:
this img tag doesn't self-close
```suggestion
    <img src="./assets/book-cover.png" class="book-cover" />
```